test/libyul/objectCompiler/wasm/no_main_function.yul

```diff
@@ -0,0 +1,22 @@
+{
+  function not_main() {
+    i64.drop(i64.add(0, 1))
+  }
+}
```

Comment:
These files were copied from `test/libyul/objectCompiler/*`. They follow the generic style of the tests.

I don't care what `.editorconfig` is doing as it is not enforced, but feel free to adjust and the files :)

Reply:
Oh, I assumed they were just brand new tests. I pointed it out specifically because `.editorconfig` is not enforced - the only way to keep the style consistent currently is to catch these things when they get added.

Also, as above, it's just a suggestion. I'd accept the PR even as is. Anyway, that's an easy change so I don't mind adjusting it myself. I'll push a commit in a moment.

Reply:
Done. Also rebased on current `develop`.